Lib/fontmake/instantiator.py

```diff
@@ -293,7 +293,7 @@ def from_designspace(
                 glyph_mutators[glyph_name] = Variator.from_masters(items, axis_order)
             except varLib.errors.VarLibError as e:
                 raise InstantiatorError(
-                    f"Cannot set up glyph '{glyph_name}' for interpolation: {e}'"
+                    f"Cannot set up glyph {glyph_name} for interpolation: {e}'"
```

Comment:
we lost the quotes around glyph_name. Also, looks like we left a trailing `'` at the end for some reasons, probably a typo

```diff
+ f"Cannot set up glyph {glyph_name} for interpolation: {e}'"
- f"Cannot set up glyph {glyph_name!r} for interpolation: {e}"
```